packages/cadl-rlc-test/test/multiclient/spec/main.cadl

```diff
@@ -0,0 +1,84 @@
+import "@cadl-lang/rest";
+import "@azure-tools/cadl-azure-core";
+import "@azure-tools/cadl-dpg";
+
+using Cadl.Http;
+using Cadl.Rest;
+using Azure.Core;
+using Azure.DPG;
+
+@service({
+  title: "Response",
+  version: "1.0.0",
+})
+namespace Cadl.Response;
+
+model Resource {
+  @visibility("read")
+  id: string;
+
+  @key
+  @segment("resources")
+  name: string;
+  description?: string;
+  type: string;
+}
+
+model ResourceArray is Resource[];
+
+// operation using Azure.Core
+@Azure.DPG.client({
```

Comment:
Do you know the difference between the dpg clien and core client?

https://github.com/Azure/cadl-azure/blob/main/packages/cadl-azure-core/src/decorators.ts#L56-L60

Because I notice that ADP uses the latter one

Reply:
the @client from core client has already been deprecated in the latest cadl-azure-core.